entrypoint.sh

```diff
@@ -4,8 +4,11 @@ if [ "$(ls -A /home/firefox/.fonts/)" ]; then
   fc-cache -f -v
 fi
 
+echo "user_pref(\"marionette.port\", $MARIONETTE_PORT);"
+echo "user_pref(\"marionette.port\", $MARIONETTE_PORT);" >> /home/firefox/profile/prefs.js
+
 ip=$(hostname --ip-address)
 
-socat tcp-listen:2828,bind="$ip",fork tcp:127.0.0.1:2828 &
+socat tcp-listen:$MARIONETTE_PORT,bind="$ip",fork tcp:127.0.0.1:$MARIONETTE_PORT &
```

Comment:
I saw you had changes for default port first but then removed it. I'd like to keep it `2828` by default and have a way to overwrite it only if it's needed. thanks!